c# working example

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Web;
+
+
+namespace GetUrl
+{
+	internal class Program
+	{
+
+		private static void Main(string[] args)
+		{
+
+
+
+
+
+
+			String accessFilters = "{}"; // converted to JSON Object of Objects
+			// String accessFilters = ("{\"advantage_sqls_mart\":{\"view_name.field_name\": \"'Your Value'\"}}");
+			         
+			String embedURL = "/embed/looks/8";
+
+			String externalUserID = "\"1\""; // converted to JSON string
+			String firstName = "\"Test\""; // converted to JSON string
+			String lastName = "\"User\""; // converted to JSON string
+			String forceLoginLogout = "true"; // converted to JSON bool
+      
+			String host = "looker.quaero.com";
```

Comment:
@haarthi maybe remove the quaero references and quaero-specific model references.